pastas/plots.py

```diff
@@ -619,14 +627,18 @@ def __init__(self, ml, tmin=None, tmax=None, update_iter=None):
 
         # calculate RMSE residuals
         res = self._residuals(self.ml.parameters.initial.values)
-        r_rmse = np.sqrt(np.sum(res ** 2))
+        r_rmse = np.sqrt(np.mean(res ** 2))
```

Comment:
Not better to call pastas.stats.metrics.rmse(res=res) ?

Reply:
Agreed, but let's do that in the next version. Can you create a GH Issue for this?

Reply:
This predates the changes to the metrics, they used to be calculated in the model methods, which could not be accessed with the callback in solve. But there is no reason not to change it now, so good idea to open a new issue.